install-npm/action.yaml

```
@@ -0,0 +1,48 @@
+name: "JavaScript Installer [npm]"
+author: "Jaspar Stach <jaspar.stach@greenbone.net>"
+description: "An action to install Greenbone JavaScript Projects"
+inputs:
+  version:
+    description: "Node version that should be installed"
+    default: "16.x"
+  token:
+    description: "Token with GitHub Packages read rights required to install."
+    required: true
```

Maybe add the default?
```suggestion
    required: true
    default: ${{ github.token }}
```